src/domain/Change.ts

```diff
@@ -0,0 +1,43 @@
+import { COINS_REVERSE } from '../constants';
+
+class Change {
+  500: number;
+  100: number;
+  50: number;
+  10: number;
+
+  constructor() {
+    this[500] = 0;
+    this[100] = 0;
+    this[50] = 0;
+    this[10] = 0;
+  }
```

Comment:
```suggestion
  500 = 0;
  100 = 0;
  50 = 0;
  10 = 0;
```
0으로 이렇게 초기화해도 number타입으로 들어가요~

Reply:
반영 링크입니다! [1a955ea](https://github.com/woowacourse/javascript-vendingmachine/pull/56/commits/1a955eaf40bb12d4565d5cd85cf44d7f580c8a5c)